network/p2p/p2p_validation_test.go

```diff
@@ -35,7 +35,7 @@ import (
 func TestP2pNetwork_MessageValidation(t *testing.T) {
 	const (
 		nodeCount      = 4
-		validatorCount = 20
+		validatorCount = 1
```

Comment:
Same here, we use `spectestingutils.Testing4SharesSet()` there only one PK. We use validator storage now, so to have 20 we need to create shares and store. I can try to mock it, if 20 is critical, but looks like with one we test all the logic.

Reply:
@moshe-blox @y0sher

Reply:
IMO its valuable to test with more keys as some P2P and validation functionality is based on validator pubkeys and their committees (subnets/topics and such).
but I think this can also be done later in separate PR .